topics/single-cell/tutorials/scrna-preprocessing-tenx/workflows/scRNA-seq-Preprocessing-TenX.ga

```diff
@@ -0,0 +1,457 @@
+{
+    "a_galaxy_workflow": "true",
+    "annotation": "",
+    "creator": [
+        {
+            "class": "Organization",
+            "identifier": "0000-0002-5192-126X",
+            "name": "Pavankumar Videm"
+        }
+    ],
+    "format-version": "0.1",
+    "license": "CC-BY-4.0",
```

Comment:
@nomadscientist are you ok with this license for this workflow?

Reply:
Yes!

Reply:
CC By for everything!